README.md

```diff
@@ -17,9 +17,11 @@ This repository hosts LivePerson’s Developer Center, which can be found at [de
 
 ## Updating the documentation
 
-All pages on the site correspond to a Markdown file (.md) which can be found inside `pages/Documents`. To update a file, please branch off of the `master` branch, edit the file in question, and create a pull request back to the master branch.
+In general, please follow the [content and code guidelines](#notes-on-content-and-code), and keep in mind [code](https://google.github.io/eng-practices/review/reviewer/standard.html) [review](https://rewind.com/blog/best-practices-for-reviewing-pull-requests-in-github/) [best](https://medium.com/yemeksepeti-teknoloji/good-manners-of-a-pull-request-some-best-practices-cb2de3c3aea1) [practices](https://blog.palantir.com/code-review-best-practices-19e02780015f).
+
+Every change needs review by a member of LivePerson’s DX team. To get a review, add or mention [j9t](), [itay1313](https://github.com/itay1313), or [lkisoslive](https://github.com/lkisoslive) as a reviewer. The review request will then be handled directly, or handed over to another DX team member. The DX team works with an [SLA](https://en.wikipedia.org/wiki/Service-level_agreement) of responding within one work day. If there is particular urgency, coordinate with the DX team ahead of time (required for requests for non-work days), aim to grant as much time as possible, and share context. If you get no response to a request after waiting for one work day, email dx-lp@liveperson.com, including a link to your pull request.
```

Comment:
there is no link for you '[j9t]()', is this in purpose, anyway this leads to a 404

Reply:
Good catch :) Added. Thanks!